src/main/java/io/confluent/connect/jdbc/source/NoOpTableMonitorThread.java

```diff
@@ -0,0 +1,45 @@
+package io.confluent.connect.jdbc.source;
+
+import io.confluent.connect.jdbc.dialect.DatabaseDialect;
+import io.confluent.connect.jdbc.util.ConnectionProvider;
+import io.confluent.connect.jdbc.util.TableId;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicBoolean;
+import org.apache.kafka.connect.connector.ConnectorContext;
+
+public class NoOpTableMonitorThread extends TableMonitorThread {
+
+    AtomicBoolean running = new AtomicBoolean(true);
+
+    public NoOpTableMonitorThread(DatabaseDialect dialect,
+                                  ConnectionProvider connectionProvider,
+                                  ConnectorContext context,
+                                  long pollMs,
+                                  Set<String> whitelist,
+                                  Set<String> blacklist) {
+        super(dialect, connectionProvider, context, pollMs, whitelist, blacklist);
+    }
+
+    @Override
+    public void run() {
+        while (running.get()) {
+            try {
+                Thread.sleep(100);
+            } catch (InterruptedException e) {
+                break;
```

Comment:
Is it possible that there are some occasional interrupts that should not terminate the thread?

Reply:
I think the only reason we'd get an interrupted exception is that the program received SIG_INT and therefore would be shutting down. At least, that's what I took away from this [blog post](https://www.yegor256.com/2015/10/20/interrupted-exception.html)